fearless/Modules/Staking/StakingMain/StakingMainPresenter.swift

```diff
@@ -24,6 +25,16 @@ final class StakingMainPresenter {
         self.logger = logger
     }
 
+    private func provideChain() {
+        guard let viewModelFactory = networkInfoViewModelFactory else {
+            return
+        }
+
+        let chainModel = viewModelFactory.createChainViewModel()
```

Comment:
Let's just pass chain to view as it is and allow view to decide how to display. Because creating view model from a single title in a separate factory looks like overengineering.

Reply:
Well, actually I was going to use the factory to provide network info data anyway. Wouldn't it be nice to keep all the methods inside?

Reply:
Got it. Thanks